framework/wazuh/core/configuration.py

```diff
@@ -769,8 +769,36 @@ def upload_group_file(group_id, file_data, file_name='agent.conf'):
 
 
 def get_active_configuration(agent_id, component, configuration):
-    """
-    Reads agent loaded configuration in memory
+    """Get active configuration in a specific node.
+
+    Parameters
+    ----------
+    agent_id : str
+        Agent ID. All possible values from 000 onwards.
+    component : str
+        Selected agent's component.
+    configuration : str
+        Configuration to get, written on disk.
+
+    Returns
+    -------
+    str
+        The active configuration the agent is currently using.
+
+    Raises
+    ------
+    WazuhError
+        If the component or configuration are not specified.
+    WazuhError
+        If the specified component is not valid.
+    WazuhError
+        If the component is not properly configured.
+    WazuhInternalError
+        If the socket cant be created.
+    WazuhInternalError
+        If the socket is not able to receive a response.
+    WazuhError
+        If the reply from the node contains an error.
```

Comment:
It is great that you added this. However, this section lacks detail. A good example: https://github.com/wazuh/wazuh/blob/edcfe6179fe86af0cb003cf9a1ef21b4e8a6480d/framework/wazuh/agent.py#L197-L204